blueprints/banking_blueprint/domains/accounts_creditcards/transfer_balances/train.txt

```diff
@@ -0,0 +1,74 @@
+Transfer {100k|sys_amount-of-money} from {checking|account_type|origin} to {savings|account_type|dest}
+Move {100k|sys_amount-of-money} to {savings|account_type|dest} from{checking|account_type|origin}
```

Comment:
Move {100k|sys_amount-of-money} to {savings|account_type|dest} from {checking|account_type|origin}